keyboards/annepro2/annepro2.c

```diff
@@ -99,6 +101,8 @@ void keyboard_post_init_kb(void) {
 
     annepro2LedGetStatus();
 
+    annepro2LedEnable();
```

Comment:
Not a fan of the casing, but that's mostly a nitpick.

Reply:
I agree,  maybe a good PR would be to standardize the casing in this keyboard.